src/components/partials/tables/FTable.vue

```diff
@@ -0,0 +1,242 @@
+                        :class="tabIndicatorColor === 'transparent' ? 'pa-xs' : ''"
+                        inline-label
+                        no-caps
+                        stretch
+                        align="left"
+                    >
+                        <q-tab
+                            v-for="tab in tabs"
+                            :key="tab.name"
+                            v-bind="tab"
+                            :name="tab.name"
+                            content-class="tab-content-class"
+                            class="text-weight-regular q-mx-sm"
+                            label=""
+                        >
+                            <span v-text="tab.label" />
+                            <q-badge
+                                v-if="tab.badge"
+                                class="q-ml-xs"
+                                v-text="tab.badge"/>
+                        </q-tab>
+                    </q-tabs>
+                </slot>
+            </div>
+            <div class="col-xs-12 col-md-6 border-bottom-grey gt-sm">
+                <div class="float-right q-pr-md">
+                    <slot name="control-right" />
+                </div>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+
+                <q-tab-panels
+                    v-model="selectedTab"
+                    animated>
+                    <q-tab-panel
+                        v-for="tab in tabs"
+                        :key="tab.name"
+                        :name="tab.name"
+                        class="q-pa-none">
+                        <div class="row">
+                            <div class="col q-pa-md border-bottom-grey">
+                                <slot :name="'filter_' + tab.name"/>
+                            </div>
+                        </div>
+                        <slot :name="'table_' + tab.name"/>
+                    </q-tab-panel>
+
+                </q-tab-panels>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col">
+                <q-footer class="page-footer">
+                    <div
+                        class="q-px-md pagination">
+                        <slot name="paginate" />
+                    </div>
+                </q-footer>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import _ from 'lodash';
+
+    export default {
+        name: 'FTable',
+        props: {
+            pageHeader: {
+                type: Boolean,
+                default: true
+            },
+            tableTitle: {
+                type: String,
+                default: ''
+            },
+            TabsData: {
+                type: Array,
+                default: () => []
+            },
+            breadcrumbs: {
+                type: Array,
+                default: () => []
```

Comment:
this should be. 

```
default: () => ([])
```

Reply:
Hi Jonel, any reference document?

Reply:
@dvikas https://github.com/vuejs/vue/issues/1032